internal/cluster/cluster.go

```diff
@@ -1268,6 +1268,11 @@ func (m *Manager) GenerateAdditionalManifests(ctx context.Context, cluster *comm
 		return errors.Wrap(err, "failed to add disk encryption manifest")
 	}
 
+	if err := m.manifestsGeneratorAPI.AddNodeIpHint(ctx, log, cluster); err != nil {
```

Comment:
should we call this only for `SNO` ? Or at least not fail in the `AddNodeIpHint` function if the machine cidr is not set. 

I believe this is useful only for SNO, tho.

Reply:
totally right, missed it. Changing to sno

Reply:
added inside function itself